dist/tools/ethos/start_network.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+trap "cleanup" EXIT #SIGINT SIGTERM
+
+create_tap() {
+    ip tuntap add ${TAP} mode tap user ${USER}
+    sysctl -w net.ipv6.conf.${TAP}.forwarding=1
+    sysctl -w net.ipv6.conf.${TAP}.accept_ra=0
+    ip link set ${TAP} up
+    ip a a fe80::1/64 dev ${TAP}
+    ip a a fd00:dead:beef::1/128 dev lo
```

Comment:
The scripts set up routing using link-local addresses between linux and l6br. If linux doesn't have a non-link-local address assigned somewhere, it cannot talk to 6lr.
So the script set's the ULA, which will then be chosen (if no other address is "closer") to communicate with 6lr or 6lbr's global address.